libc/src/ctype/isalpha.h

```diff
@@ -9,6 +9,7 @@
 #ifndef LLVM_LIBC_SRC_CTYPE_ISALPHA_H
 #define LLVM_LIBC_SRC_CTYPE_ISALPHA_H
 
+#include "include/llvm-libc-types/locale_t.h"
```

Comment:
I don't think this include is necessary for the normal variants of each of these functions.